app/services/LocationService.js

```diff
@@ -64,21 +65,21 @@ export default class LocationServices {
             desiredAccuracy: BackgroundGeolocation.HIGH_ACCURACY,
             stationaryRadius: 50,
             distanceFilter: 50,
-            notificationTitle: 'PrivateKit Enabled',
-            notificationText: 'PrivateKit is recording path information on this device.',
+            notificationTitle: 'Private Kit Enabled',
+            notificationText: 'Private Kit is securely storing your GPS coordinates once every five minutes on this device.',
```

Comment:
Might not say securely storing as we are not encrypting the coordinates, they are stored in raw format. However, no other app can access it so it is good as well.